src/test/sql/sql_pipeline_test.cpp

```diff
@@ -453,4 +456,64 @@ TEST_F(SQLPipelineTest, RequiresExecutionVariations) {
   EXPECT_FALSE(SQLPipeline{multi_no_exec}.requires_execution());
 }
 
+TEST_F(SQLPipelineTest, CorrectStatementStringSplitting) {
+  SQLPipeline select_pipeline{_select_query_a};
```

Comment:
Consider a short "Tests that..." docstring for the method, from a quick look and reading the name, I can't tell what it does.